### PR TITLE
Improve Docstrings for `sort` and `sort!`

### PR DESCRIPTION
**Pull request by Chandu-4444**
This is a sample PR, I enhanced the readability by adding a newline for each keyword argument.

### CODE REVIEW

File: src/abstractdataframe/sort.jl [hunk 1]
```diff
@@ -452,6 +453,7 @@ sorted order according to column(s) `cols`.
 If `alg` is `nothing` (the default), the most appropriate algorithm is
 chosen automatically among `TimSort`, `MergeSort` and `RadixSort` depending
 on the type of the sorting columns and on the number of rows in `df`.
+
 If `rev` is `true`, reverse sorting is performed. To enable reverse sorting
 only for some columns, pass `order(c, rev=true)` in `cols`, with `c` the
 corresponding column index (see example below).
```
Comment (bkamins) [inline src/abstractdataframe/sort.jl:459]:
```suggestion
corresponding column index (see example below).

```

File: src/dataframe/sort.jl [hunk 2]
```diff
@@ -4,13 +4,14 @@
...
 
 `cols` can be any column selector ($COLUMNINDEX_STR; $MULTICOLUMNINDEX_STR).
 
 If `alg` is `nothing` (the default), the most appropriate algorithm is
 chosen automatically among `TimSort`, `MergeSort` and `RadixSort` depending
 on the type of the sorting columns and on the number of rows in `df`.
+
 If `rev` is `true`, reverse sorting is performed. To enable reverse sorting
 only for some columns, pass `order(c, rev=true)` in `cols`, with `c` the
 corresponding column index (see example below).
```
Comment (bkamins) [inline src/dataframe/sort.jl:17]:
```suggestion
corresponding column index (see example below).

```

File: src/dataframe/sort.jl [hunk 3]
```diff
@@ -4,13 +4,14 @@
           alg::Union{Algorithm, Nothing}=nothing, lt=isless, by=identity,
           rev::Bool=false, order::Ordering=Forward)
 
-Sort data frame `df` by column(s) `cols`.
+Sort data frame `df` by column(s) `cols`. The sorting on multiple columns is done lexicographicallly.
```
Comment (bkamins) [inline src/dataframe/sort.jl:7]: is "The" needed before "sorting"?

Also you might add the same comment in `sort` above.

**Comment by bkamins** [comment]
@Chandu-4444 - preferably use another branch than `main` in your fork for making your PRs. The reason is that if you use `main` you do not have an easy way to update your forks `main` when origin `main` gets updated (it is best to set fork's `main` as tracking branch of origin `main` in general).

**Comment by Chandu-4444** [comment]
> @Chandu-4444 - preferably use another branch than `main` in your fork for making your PRs. The reason is that if you use `main` you do not have an easy way to update your forks `main` when origin `main` gets updated (it is best to set fork's `main` as tracking branch of origin `main` in general).

Yeah, I guess that's the reason why updating my main branch leads to another commit. Thanks, I'll do this!

File: src/dataframe/sort.jl (hunk 3 above)
Comment (Chandu-4444) [reply src/dataframe/sort.jl:7]: Yeah, that isn't needed. I'll remove this and will add the comment to the `sort`.

**Comment by Chandu-4444** [comment]
I have updated the PR with suggested changes. I'll create a new branch and will improve the docs further after this gets merged.

File: src/abstractdataframe/sort.jl [hunk 4]
```diff
@@ -364,13 +364,14 @@ end
          alg::Union{Algorithm, Nothing}=nothing, lt=isless, by=identity,
          rev::Bool=false, order::Ordering=Forward, view::Bool=false)
 
-Return a data frame containing the rows in `df` sorted by column(s) `cols`.
+Return a data frame containing the rows in `df` sorted by column(s) `cols`. Sorting on multiple columns is done lexicographically.
```
Comment (bkamins) [inline src/abstractdataframe/sort.jl:367]:
```suggestion
Return a data frame containing the rows in `df` sorted by column(s) `cols`.
Sorting on multiple columns is done lexicographically.
```

File: src/dataframe/sort.jl [hunk 5]
```diff
@@ -4,16 +4,18 @@
           alg::Union{Algorithm, Nothing}=nothing, lt=isless, by=identity,
           rev::Bool=false, order::Ordering=Forward)
 
-Sort data frame `df` by column(s) `cols`.
+Sort data frame `df` by column(s) `cols`. Sorting on multiple columns is done lexicographicallly.
```
Comment (bkamins) [inline src/dataframe/sort.jl:7]:
```suggestion
Sort data frame `df` by column(s) `cols`.
Sorting on multiple columns is done lexicographicallly.
```

**Review by bkamins — APPROVED** [review]

**Comment by bkamins** [comment]
Thank you!

**Pull request merged**